homeassistant/components/unifi/switch.py

```diff
@@ -57,9 +69,28 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             client = controller.api.clients_all[mac]
             controller.api.clients.process_raw([client.raw])
 
+    # Create update coordinator for DPI Restrictions
+    if controller.option_dpi_restrictions:
```

Comment:
This will not work if the user enables it after everything has loaded. You need to handle loading/unloading of this from the items_added function

Reply:
Since Update Coordinator refresh won't be called unless there are subscribers, I think I can remove this conditional and always create the coordinator.